src/routes/route.js

```diff
@@ -1,4 +1,8 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { fetchAllroutes } from "../redux/Action/routes";
+
 import { Routes, Route } from "react-router-dom";
 // import Dashboard from "../Components/dashboard/dashboard/Dashboard.jsx";
```

Comment:
Let's remove these commented codes if they are not needed.

Reply:
Look like I missed these ones, I have resolved them